services/minespace-web/src/styles/overrides/antd-overrides.scss

```diff
@@ -280,7 +280,6 @@ h5.ant-typography,
 }
 
 .ant-table-thead>tr>th {
-  background: #F7F8FA;
```

Comment:
This hasn't actually changed, but now it's in the webpack variables! Who knew that the index.pure.less rules I'm always fighting were defined by us all along...